js/index.js

```diff
@@ -79,15 +74,55 @@ function submitMessage(event) {
     editButton.addEventListener('click', function(){
        newMessage.innerHTML = `<a href="mailto:${email}">${username}</a>
        <textarea> ${message}</textarea>`
+
+       //Creating the secind Submit button
+       const resubmitButton = document.createElement('button');
+       resubmitButton.innerHTML = "Submit";
+       resubmitButton.setAttribute("type", "submit");
+       resubmitButton.setAttribute("class", "resubmit_button");
+       newMessage.appendChild(resubmitButton);
        newMessage.appendChild(removeButton);
-       newMessage.appendChild(editButton); 
+       
+       newMessage.addEventListener('submit', () => {
+       const message = event.target.textarea.value;
+            newMessage.innerHTML = `<a href="mailto:${email}">${username}</a>
+            <span> ${message}</span>`;
+       })
     });
 
-    newMessage.appendChild(removeButton);
+
+    newMessage.appendChild(editButton);
     messageList.appendChild(newMessage);
-    newMessage.appendChild(editButton);   
+    newMessage.appendChild(removeButton);   
 
     messageForm.reset(); //to reset the fields after submit
 }
 
-
+// Creating fetch
+fetch("https://api.github.com/users/katy1313/repos")
+.then(response => {
+    if(!response.ok) {
+        throw new Error("Not Found");
+    }
+    return response.json();
+})
+.then(repositories => {
+    if(repositories.length === 0) {
+        throw new Error("No repos exist");
+    } else {
+        console.log(repositories);
+    }  
+    //Display repositories in the list
```

Comment:
Good use of a comment here. It tells the reader what the code's doing without explaining it line by line.